bgpd/bgp_damp.c

```diff
@@ -494,7 +494,8 @@ static const char *bgp_get_reuse_time(unsigned int penalty, char *buf,
 	struct tm tm;
 	int time_store = 0;
 
-	if (penalty > damp[afi][safi].reuse_limit) {
+	if (damp[afi][safi].reuse_limit &&
```

Comment:
isn't this a bug in the setup of the damp[afi][safi]?  shouldn't reuse_limit be set to DEFAULT_REUSE or if the particular afi/safi is not setup to show this then shouldn't have bgp_damp_reuse_time_vty should have stopped this?

Reply:
this test seems broken to me:
```
	/* If dampening is not enabled or there is no dampening information,
	   return immediately.  */
	if (!bdc || !bdi)
		return NULL;```

in bgp_damp_reuse_time_vty bdc is always going to be true since it's always a valid pointer.  

we should be checking bgp->af_flags[afi][saf] for BGP_CONFIG_DAMPENING being set instead of !bdc

Reply:
bgp_damp_info_vty and bgp_damp_reuse_time_vty need to be fixed to not check !bdc and check this flag instead.

Reply:
Fixed.